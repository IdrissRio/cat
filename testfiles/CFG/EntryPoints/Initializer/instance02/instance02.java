public class instance02 { int i = 0; }
/**Testing  field initialization */