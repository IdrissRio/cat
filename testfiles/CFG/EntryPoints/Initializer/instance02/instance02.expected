NodeName: Entry[15] Successors: [Exit[16]]
NodeName: Exit[16] Successors: []
NodeName: Entry[7] Successors: [IntegerLiteral[36]]
NodeName: IntegerLiteral[36] Successors: [FieldDeclarator[33]]
NodeName: FieldDeclarator[33] Successors: [Exit[8]]
NodeName: Exit[8] Successors: []

NodeName: Exit[16] Predecessor: [Entry[15]]
NodeName: Entry[15] Predecessor: []
NodeName: Exit[8] Predecessor: [FieldDeclarator[33]]
NodeName: FieldDeclarator[33] Predecessor: [IntegerLiteral[36]]
NodeName: IntegerLiteral[36] Predecessor: [Entry[7]]
NodeName: Entry[7] Predecessor: []
